apps/tx-builder/src/components/EditableLabel.tsx

```diff
@@ -21,7 +21,7 @@ const EditableLabel = ({ children, onEdit }: EditableLabelProps) => {
 
 export default EditableLabel;
 
-const EditableComponent = styled.span`
+const EditableComponent = styled.div`
```

Comment:
It seems that this change is causing this warning:
![Captura de pantalla 2022-03-29 a las 11 14 10](https://user-images.githubusercontent.com/26763673/160577869-1f4ae19d-873a-4312-968f-1835af5dcf94.png)